src/Fabulous/View.fs

```diff
@@ -0,0 +1,41 @@
+namespace Fabulous
+
+module ViewHelpers =
+    let canReuseView (prevWidget: Widget) (currWidget: Widget) =
+        let prevKey = prevWidget.Key
+
+        if not(prevKey = currWidget.Key) then
+            false
+        else if (prevKey = Memo.MemoWidgetKey) then
+            Memo.canReuseMemoizedViewNode prevWidget currWidget
+        else
+            true
+
+module View =
+    let memo<'msg, 'key, 'marker when 'key: equality>
```

Comment:
I think it might be actually, I think it will be used sparingly across a project. Maybe 10 - 100(ish) usages at max. I think inlining it will be actually worse because it is not on the hot path and will produce more code (IL instructions).

At least that was my reasoning not to inline it

Reply:
Oh ok, I see. Yes, it won't be used extensively.